snapcraft/plugins/catkin.py

```diff
@@ -25,6 +25,9 @@
     - catkin-packages:
       (list of strings)
       List of catkin packages to build.
+    - source-space:
```

Comment:
Yeah, line 262. Workspace would be confusing since that's what `source`/`source-subdir` should be pointing to (the catkin workspace).